src/test/kotlin/org/stt/text/JiraExpansionProviderTest.kt

```diff
@@ -1,77 +1,77 @@
 package org.stt.text
 
-//import net.rcarz.jiraclient.Issue
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.After
 import org.junit.Before
 import org.junit.Test
 import org.mockito.BDDMockito.given
 import org.mockito.Mock
 import org.mockito.MockitoAnnotations
+import org.stt.connector.jira.Issue
 import org.stt.connector.jira.JiraConnector
 import java.util.*
 
 class JiraExpansionProviderTest {
 
-//    @Mock
```

Comment:
please also here ... remove the commented-out code

Reply:
It's the old version of the file. Tests comment-in and are adjusted to run successfully.

Reply:
sorry for this - was confused